### ISSUE TITLE
robot communication + volumetric mapping (kimera-semantics and kimera-PGMO)

### ISSUE THREAD

**Issue by LauGasp** [MIT-SPARK/Kimera-Multi#25]
Hello, thank you for making this project.

I'm not able to visualize the result of the pointcloud (semantic pointcloud) of the semantics module (I'm using [this](https://github.com/MIT-SPARK/Kimera-Semantics/blob/feature/kimera_multi/kimera_semantics_ros/launch/kimera_semantics_jackal.launch) launch) using the bags provided by you (e.g. 10_14_acl_jackal.bag). The problem doesn't seem to be tf at the moment, but perhaps the fact that the topics they provide (depth and color) are not aligned...(I'm not sure if this is the cause of the problem). The problem is in this node `<node pkg="nodelet" type="nodelet" name="nodelet_manager_ks" args="manager"  output="screen"/>`. 
The pointcloud output is not generated.
Can you help me solve it?

I think my problem is very much in line with these unresolved  #21 #24

Basically, I want to do an individual volumetric reconstruction of each robot before doing it with all the robots. 
The image of this mapping appears on the home page...
![imagem](https://github.com/MIT-SPARK/Kimera-Multi/assets/159635376/01625fbb-3a83-4fd5-bdf3-c87159a49733)

Note: I can see the path of each robot (from the VIO module and from dpgo). 

Another question, "Inter-robot communication is performed in a fully peer-to-peer manner using a lightweight communication layer on top of the UDP protocol using the Remote Topic Manager." This information is in your readme. Is it not available yet? If I want to test with 2 robots, how would I communicate?

**Comment by South-River** [comment]
i think maybe you can try my repo https://github.com/South-River/swarm_bridge for robot's communication. it uses udp for autonomous connection establish, and using tcp for robust data transmission.

**Comment by LauGasp** [comment]
> i think maybe you can try my repo https://github.com/South-River/swarm_bridge for robot's communication. it uses udp for autonomous connection establish, and using tcp for robust data transmission.

Hello! I’m not needing it anymore, but thank you very much! It could always be useful for someone. Great work!!
